hm8-1.py

```diff
@@ -0,0 +1,28 @@
+# 1 Написати власний декоратор, задачею якого має бути друк назви функції і часу, коли вона була викликана.
+# Декоратор має працювати для різних функцій однаково.
+# importing library
+import time
+import math
+
+
+# decorator to calculate curent time when function is called
+def calculate_time(func):
+    def inner1(*args, **kwargs):
+        # storing time before function execution
+        begin = time.localtime()
+
+        func(*args, **kwargs)
```

Comment:
Save the result of this func call to some variable and return the variable after the last print. It will allow you to decorate functions that return some value